src/Platform/Microsoft.Testing.Platform.MSBuild/buildMultiTargeting/Microsoft.Testing.Platform.MSBuild.props

```diff
@@ -8,8 +8,8 @@
   <!-- Register the PlatformOutputDevice extension -->
   <ItemGroup>
     <TestingPlatformBuilderHook Include="8214564B-6107-422D-8A82-462F97602287" >
-      <DisplayName>Microsoft.Testing.Platform.MSBuild</DisplayName>
-      <TypeFullName>Microsoft.Testing.Platform.MSBuild.TestingPlatformBuilderHook</TypeFullName>
+      <DisplayName>Microsoft.Testing.Extensions.MSBuild</DisplayName>
+      <TypeFullName>Microsoft.Testing.Extensions.MSBuild.TestingPlatformBuilderHook</TypeFullName>
```

Comment:
Do we prefer to keep the Platform namespace?

Reply:
I would keep platform, it's not an extension but a plug to msbuild, now we will split a dll as extension.

Reply:
That's ok, I need to change the namespace of the hook c# class inside the extension dll then.